src/main/java/uk/gov/hmcts/reform/ccd/service/remote/clients/LauClient.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.ccd.service.remote.clients;
+
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import uk.gov.hmcts.reform.ccd.data.lau.CaseActionPostRequestResponse;
+
+import static uk.gov.hmcts.reform.ccd.util.RestConstants.LAU_SAVE_PATH;
+
+@FeignClient(name = "lauClient", url = "${ccd.log.and.audit.host}")
```

Comment:
A bit pedantic note - but is it really ccd? and I feel like it would look and read better if it had been `log-and-audit`, or `log-audit`, or even `logAndAudit`, rather than separated by dots.